### PR TITLE
chore: add fd-theming mapping

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#1164

## Description
This pr will create a map for the old fundamental theming variables to the new ones of fiori 3
## Screenshots
NONE

#### Please check whether the PR fulfills the following requirements

- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-styles/wiki/PR-Review-Checklist

### CODE REVIEW

File: src/mixins/_mixins.scss [hunk 1]
```diff
@@ -343,3 +343,66 @@
...
+  --fd-color-text-1: var(--sapTextColor, #32363a),
+  --fd-color-text-2: var(--sapTextColor, #515559),
+  --fd-color-text-3: var(--sapContent_LabelColor, #6a6d70),
+  --fd-color-text-4: var(--sapfield_PlaceholderTextColor, #74777a),
+  --fd-color-text-5: var(--sapContent_Selected_TextColor, #ffffff),
+  --fd-color-background-1: var(--sapShell_Background, #edeff0),
+  --fd-color-background-2: var(--sapBaseColor, #ffffff),
+  --fd-color-background-3: var(--sapSuccessBackground, #f1fdf6),
+  --fd-color-background-4: var(--sapWarningBackground, #fef7f1),
+  --fd-color-background-5: var(--saperrorBackground, #ffebeb),
```
Comment (InnaAtanasova) [inline src/mixins/_mixins.scss:365]: Typo `--sapErrorBackground`

File: src/mixins/_mixins.scss [hunk 2]
```diff
@@ -343,3 +343,66 @@
...
+  --fd-color-background-6: var(--sapInformationBackground, #f5faff),
+  --fd-color-background-7: var(--sapNeutralBackground, #f4f4f4),
+  --fd-color-neutral-1: var(--sapList_footerBackground, #fafafa),
```
Comment (InnaAtanasova) [inline src/mixins/_mixins.scss:368]: Typo `--sapList_FooterBackground`

File: src/mixins/_mixins.scss [hunk 3]
```diff
@@ -343,3 +343,66 @@
...
+  --fd-color-accent-7: var(--sapAccentColor7, #0f828f),
+  --fd-color-accent-8: var(--sapAccentColor8, #7ca10c),
+  --fd-color-accent-9: var(--sapAccentColor9, #955ecf),
+  --fd-color-accent-10: var(--sapAccentColor10, #647887),
+  --fd-color-accent-11: var(--sapAccentColor1, #d17f15),
+  --fd-color-accent-12: var(--sapAccentColor2, #d04343),
+  --fd-color-accent-13: var(--sapAccentColor6, #2b78c5),
+  --fd-color-accent-14: var(--sapAccentColor9, #0f828f),
+  --fd-color-accent-15: var(--sapAccentColor4, #647887),
+  --fd-color-action-hover: var(--sapButton_emphasized_Hover_Background, #085caf),
```
Comment (InnaAtanasova) [inline src/mixins/_mixins.scss:394]: Typo `--sapButton_Emphasized_Hover_Background`

File: src/mixins/_mixins.scss [hunk 4]
```diff
@@ -343,3 +343,66 @@
...
+  --fd-color-action-3: var(--sapButton_Selected_Background, #0854a1),
+  --fd-color-shell-1: var(--sapShellColor, #354a5f),
+  --fd-color-shell-2: var(--sapShell_InteractiveTextColor, #d1e8ff),
+  --fd-color-shell-3: var(--sapShell_Hover_Background, #283848),
+  --fd-color-shell-4: var(--sapShell_Active_Background, #23303e),
+  --fd-color-shell-5: var(--sapShell_InteractiveBorderColor, #7996b4),
+  --fd-color-text-1: var(--sapTextColor, #32363a),
+  --fd-color-text-2: var(--sapTextColor, #515559),
+  --fd-color-text-3: var(--sapContent_LabelColor, #6a6d70),
+  --fd-color-text-4: var(--sapfield_PlaceholderTextColor, #74777a),
```
Comment (InnaAtanasova) [inline src/mixins/_mixins.scss:359]: Typo `--sapField_PlaceholderTextColor`

**Comment by InnaAtanasova** [comment]
LGTM

**Comment by droshev** [comment]
@stefanoScalzo let's not merge this as we discussed

**Pull request closed without merging**